Cargo.toml

```diff
@@ -70,6 +70,22 @@ disable_target_static_assertions = [
 ]
 # If this is active all the code compiles in synchronous mode. Otherwise `async` is supported.
 sync-api = ["ac-compose-macros/sync-api", "maybe-async/is_sync"]
+# Use the `jsonrpsee` crate for websocket communication. Does provide sync and async support but needs a tokio runtime.
+# Provides convenience functions such as subscription callbacks.
+jsonrpsee-client = ["std", "jsonrpsee", "futures"]
+# Use the `tungstenite` crate for websocket communication. No async support but has some reconnection capabilities.
+tungstenite-client = ["std", "tungstenite"]
+# Use the `ws` crate for websocket communication. No async support.
+# Establishes a new connection for each request and therefore is limited in terms of performance.
```

Comment:
Cool! I like this documentation very much. Your example linking below just gave me the idea: Would it make sense to link to the ws_client example as well? (same for tungestenite)

Reply:
Makes sense. I added the examples to the documentation.